content/blog/2020/2020-04-02-covid-19-basic.md

```diff
@@ -0,0 +1,205 @@
+
+Easier still, Orange's File widget can load the data directly from the web, and handles a basic .csv file (for more fine-grained options, use CSV File Import). So, for starters, add the File widget to the canvas and copy the above link to the URL field. You can then connect it to a Data Table widget and check that everything's loaded OK. You should see a table with rows corresponding to regions and countries, and columns corresponding to dates, with two additional columns detailing region locations (latitude, longitude).
+
+Note that this is live data, so your results will differ from those we show here.
+
+## Plot the usual plots
+
+Connect the File to a Line Plot widget to see the graph. By default, Line Plot shows means and ranges, while we are interested in raw lines. Let's click the checkboxes accordingly.
+
+{{% figure src="/blog_img/2020/2020-04-02-lineplot.png" %}}
+
+Pretty boring. Anybody can do this in Excel. Orange allows us to play with these curves, though.
+
+The curve that grows rapidly but then flattens out -- is probably China, right? To check this, select it. It may be hard to click the curve, so select it by dragging across it. Then, connect the Line Plot widget to Data Table, and it will show the data for this curve. You'll learn that this is, of course, the Hubei province – where it all started.
+
+What are the fast-rising curves? Select them by dragging a line across them, and check the Data Table widget: these are the US, Iran, Italy, Spain, Germany and France.
+
+We can do it the other way around: connect File to Data Table and Data Table to Line Plot, such that the Data Table sends its selected subset of data.
+
+{{% figure src="/blog_img/2020/2020-04-02-workflow-select-w-table.png" %}}
+
+In the Data Table, you can now find and click your favourite country. Line Plot will now highlight the curve for Slovenia (or whichever country you chose). You can also select multiple countries, like all the Chinese provinces. Or all European countries. Well, the latter is a bit difficult; there is no data about the continent. There's a trick, though. Replace the Data Table widget with a Scatter Plot widget, and choose Long and Lat for x and y respectively. Zoom in on Europe and select the points by dragging across them. Line Plot will highlight the curves belonging to the countries chosen in Scatter Plot.
+
+In the case of Slovenia (and possibly your choise as well), we couldn't really see its curve because it's negligent in comparison with others. The only thing we've learned is that the 327-million-man US have more cases of COVID-19 than a 2-million-man Slovenia. The same goes for Chinese provinces: the only one that sticks out is Hubei. Also, the "map" we improvised in Scatter Plot was a poor-man's map. Orange has proper geographical support (Orange3-Geo add-on), we'll look at in-depth in a follow-up post.
```

Comment:
choise --> choice

Reply:
we'll look at in-depth in a follow-up post --> we'll look at it in-depth in a follow-up post